src/middlewared/middlewared/plugins/keychain.py

```diff
@@ -464,8 +488,12 @@ async def get_of_type(self, id_, type_):
 
             return credential
 
-    @api_method(KeychainCredentialGenerateSSHKeyPairArgs, KeychainCredentialGenerateSSHKeyPairResult,
-                roles=["KEYCHAIN_CREDENTIAL_WRITE"])
+    @api_method(
+        KeychainCredentialGenerateSSHKeyPairArgs,
+        KeychainCredentialGenerateSSHKeyPairResult,
+        roles=["KEYCHAIN_CREDENTIAL_WRITE"],
+        audit="Generate SSH Key Pair"
```

Comment:
Same here, what's interesting about generating random numbers that needs to be logged?

Reply:
I agree